src/index.json

```diff
@@ -975,6 +975,53 @@
                     "version": "0.1.0"
                 }
             }
+        ],
+        "hanaonazure": [
```

Comment:
Change this extension name to something like "sap-hana". No need to refer to "azure" since for the most part, the services included in the CLI/extensions are part of azure.

Reply:
changed to saphana